src/apim.runtime.module.ts

```diff
@@ -94,7 +94,7 @@ export class ApimRuntimeModule implements IInjectorModule {
         injector.bind("graphqlConsole", GraphqlConsole);
         injector.bind("authorization", Authorization);
         injector.bind("typeDefinition", TypeDefinitionViewModel);
-        injector.bind("codeSample", CodeSampleViewModel);
+        injector.bind("codeSnippet", CodeSnippet);
```

Comment:
Shouldn't we keep the naming convention and name this class `CodeSnippetViewModel` ? 
Or should we agree that we drop the `ViewModel` naming and make this uniform among the application?

Reply:
There is no actual naming convention in place, but that's a good point to have one. I have added a work item for us to address it later.